webextensions/api/storage.json

```diff
@@ -372,7 +372,8 @@
               "firefox_android": "mirror",
               "opera": "mirror",
               "safari": {
-                "version_added": false
+                "version_added": "16.4",
+                "notes": "Maximum allowed storage is 10MB."
```

Comment:
@jdatapple I don't seem to be able to find any Apple documentation that confirms this. The information on [Review your web extension API usage](https://developer.apple.com/documentation/safariservices/safari_web_extensions/assessing_your_safari_web_extension_s_browser_compatibility#3584139) seems to suggest that `unlimited` grants unlimited storage.

Reply:
The storage.session API is distinct from the `storage.local` (and `storage.sync` and `storage.managed`) APIs. They have different storage backends and different quotas.

Safari has shipped, with 10 MB as a quota. This is noteworthy because the only other implementation (Chromium) used to have 1 MB as a limit.

Recently, there was consensus among browsers to bump the limit from 1 MB to 10 MB: https://github.com/w3c/webextensions/issues/350

So I suggest the following:
- Remove the `notes` from Safari.
- Modify the `chrome` support to note that before 112, the limit was 1 MB. (starting from Chrome 112, the limit is 10 MB).

Update mdn/content at https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/session to correct the statement about the quota (it currently states 1 MB).

Reference for Chrome bug that bumped the limit to 10 MB:
https://bugs.chromium.org/p/chromium/issues/detail?id=1385167

Reply:
I made the note changes in [31e6d1d](https://github.com/mdn/browser-compat-data/pull/18997/commits/31e6d1df6c5eb8764c9c8b1396fad58d29662882).

Reply:
I also submitted a PR for the content changes on MDN. See [#24853](https://github.com/mdn/content/pull/24853).